eZ/Publish/API/Repository/Tests/Values/Content/DefaultLanguageCodeForContentTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Content;
```

Comment:
This namespace is incorrect. Moreover new classes should be placed in the new structure, so `test/` directory.

Reply:
I don't know how this happened O.O
I blame PHPStorm over Jetbrains Gateway